docs/modules/ROOT/pages/nre.adoc

```diff
@@ -214,3 +179,208 @@ Address of the contract to query.
 - `*balance*`
 +
 Balance of the contract.
+
+== Account API
+
+Public API of the Account abstraction.
+
+=== `send`
+
+[.contract-item]
+[[send]]
+==== `[.contract-item-name]#++async send++#++(self, address_or_alias, method, calldata, nonce=None, max_fee=None) → transaction++`
+
+Return a Transaction instance representing an invoke transaction.
+
+===== Arguments
+
+- `*address_or_alias*`
++
+Target contract identifier (alias needs to be registered in deployments).
+- `*method*`
++
+Method to execute.
+- `*calldata*`
++
+Arguments for the call.
+- `*nonce*`
++
+Account nonce. Is automatically computed when is left as `None`.
+- `*max_fee*`
++
+The max fee you are willing to pay for the transaction execution.
++
+This value will be usually left as `None`, because
+the returned transaction allows to estimate and update the fee later.
+
+===== Return values
+
+- `*transaction*`
++
+A link:#transaction_api[Transaction] instance.
+
+=== `declare`
+
+[.contract-item]
+[[declare]]
+==== `[.contract-item-name]#++async declare++#++(self, contract_name,  nonce=None, max_fee=None, alias=None, overriding_path=None, nile_account=False) → transaction++`
+
+Return a Transaction instance representing a declare transaction.
+
+===== Arguments
+
+- `*contract_name*`
++
+The name of the contract to declare (for artifacts resolution).
+- `*nonce*`
++
+Account nonce. Is automatically computed when is left as `None`.
+- `*max_fee*`
++
+The max fee you are willing to pay for the transaction execution.
++
+This value will be usually left as `None`, because
+the returned transaction allows to estimate and update the fee later.
+- `*alias*`
++
+The alias for registering the declared class_hash (DEPRECATED).
+- `*overriding_path*`
++
+Path override for artifacts and abi resolution.
+- `*nile_account*`
++
+Wheter to use the OZ account artifact.
+
+===== Return values
+
+- `*transaction*`
++
+A link:#transaction_api[Transaction] instance.
+
+=== `deploy_contract`
+
+[.contract-item]
+[[deploy_contract]]
+==== `[.contract-item-name]#++async deploy_contract++#++(self, contract_name, salt, unique, calldata, nonce=None, max_fee=None, deployer_address=None, alias=None, overriding_path=None, abi=None) → transaction++`
```

Comment:
I think we should add it for sure, but after separating the Account initialization from deployment. For now, users don't need to call this deploy method directly even in NRE. There is just a `get_or_deploy_account` method they can use, and the returned account is deployed. 

If we add it now, it could be kind of confusing, what do you think?

Reply:
Mmm right. Yeah, I agree. I'll leave a comment on the account initialization issue to remind ourselves to update the API